Uplift/ViewModels/MainViewModel.swift

```diff
@@ -55,8 +68,8 @@ extension MainView {
             // TODO: Update for next giveaway
             Network.client.mutationPublisher(
                 mutation: CreateUserMutation(
-                    email: "",
-                    name: "",
+                    email: self.email,
+                    name: self.name,
```

Comment:
Since the actual database itself has not been updated to store emails or names, I think it doesn't make a difference whether we use empty string or `self.email ` because both are just Strings.